src/sbvr-api/permissions.ts

```diff
@@ -1252,6 +1252,16 @@ const $getApiKeyPermissions = (() => {
 												$alias: 'k',
 												$expr: {
 													k: { key: { '@': 'apiKey' } },
+													$or: [
+														{
+															k: { expiry_date: null },
```

Comment:
Did I ever submitted my review about changing the `khp` to `k` here?
B/c I can't find it in the GH comments atm !?!?!?

Reply:
My initial changes had that when the `$or` was a level above. Is there something else to add/update?